docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,112 @@
+Migrating Your Existing Workflows to Use GitHub Apps
+======================================================
+
+Dockstore 1.9.0 provides users with a way to keep their workflows automatically updated (instead of needing to manually refresh) by using GitHub apps.
+Here, we will go over how to migrate your existing Dockstore workflows to use GitHub apps. Also keep in mind that the migration process is done one branch/version at a time.
+This tutorial assumes that you have read both our overview of :doc:`GitHub apps </getting-started/github-apps>`. and the :doc:`GitHub apps section </getting-started/dockstore-workflows>`
```

Comment:
The text for the 2nd link is 'Github apps section' but its actually going to the overall page for the workflow registration documentation. For now this is not that big of a deal, since the section is near the top, but might not be the case in the future. To resolve, I think either update existing to link directly to the github apps section or if keeping the link the same, add it to the 'workflow registration documentation' text instead.